registry/storage/driver/testsuites/testsuites.go

```diff
@@ -3,6 +3,10 @@ package testsuites
 import (
 	"bytes"
 	"crypto/sha1"
+	"github.com/docker/distribution/context"
```

Comment:
@stevvooe, when i add the digest package in testsuites.go, and save the change, seems that fmt changed, this, i will check it. Thanks.